### PR TITLE
Update quarantine-email-messages.md

### PR DESCRIPTION
**Pull request by ryancairnes**
Tweaking the language slightly to reflect the behavior.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/quarantine-email-messages.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ By default, messages that were filtered for phishing, malware, and mail flow rul
 Both users and admins can work with quarantined messages. Users can work with just their own filtered messages in quarantine. Admins can search for and manage quarantined messages for all users.
 
 > [!NOTE]
-> High confidence phish messages and messages quarantined by mail flow rule actions are only available in the admin quarantine. Users can access their own phish, spam, and bulk mail messages. 
+> Malware, high confidence phish messages and messages quarantined by mail flow rule actions are only available in the admin quarantine. Users can access their own phish, spam, and bulk mail messages.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/quarantine-email-messages.md:32]:
```suggestion
> Malware, high confidence phish messages, and messages quarantined by mail flow rule actions are only available in the admin quarantine. Users can access their own phish, spam, and bulk mail messages. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@ryancairnes  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by vern7509 — APPROVED** [review]

**Pull request merged**